src/main/java/com/dabsquared/gitlabjenkins/connection/GitLabConnectionConfig.java

```diff
@@ -64,7 +63,7 @@ public boolean isUseAuthenticatedEndpoint() {
         return useAuthenticatedEndpoint;
     }
 
-    void setUseAuthenticatedEndpoint(boolean useAuthenticatedEndpoint) {
+    public void setUseAuthenticatedEndpoint(boolean useAuthenticatedEndpoint) {
```

Comment:
Usual JCasC compatibility practice is to use `@DataBoundSetter` and `@DataBoundConstructor` for global configurations